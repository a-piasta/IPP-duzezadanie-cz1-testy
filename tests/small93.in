extendRoute;50;ltokorrvki
addRoad;tdlnytdleb;tdlnytdleb;0;-1
addRoad;xvekfjqttaewpxkkiocdhhqw;uzdsgvzxsdgfrpmzezzhkst;21;14
removeRoad;xvekfjqttaewpxkkiocdhhqw;uzdsgvzxsdgfrpmzezzhkst
addRoad;qmtshjwkvxrwhhnivgqfpxbj;ltokorrvki;15;9
getRouteDescription;334
extendRoute;50;uzdsgvzxsdgfrpmzezzhkst
newRoute;710;tdlnytdleb;uzdsgvzxsdgfrpmzezzhkst
repairRoad;tdlnytdleb;hnsxaeswfw;2
addRoad;ltokorrvki;loimvtfqvodpmfveb;13;13
newRoute;500;jatndbmbfzi;qmtshjwkvxrwhhnivgqfpxbj
newRoute;867;ltokorrvki;qmtshjwkvxrwhhnivgqfpxbj
repairRoad;tdlnytdleb;qmtshjwkvxrwhhnivgqfpxbj;7
extendRoute;710;tdlnytdleb
addRoad;xvekfjqttaewpxkkiocdhhqw;oifinnybjrkspupolm;25;7
newRoute;520;ltokorrvki;ltokorrvki
repairRoad;xvekfjqttaewpxkkiocdhhqw;tdlnytdleb;5
addRoad;hnsxaeswfw;tdlnytdleb;7;-13
addRoad;oifinnybjrkspupolm;tdlnytdleb;16;-8
repairRoad;uzdsgvzxsdgfrpmzezzhkst;loimvtfqvodpmfveb;-14
getRouteDescription;710
extendRoute;867;uzdsgvzxsdgfrpmzezzhkst
extendRoute;520;uzdsgvzxsdgfrpmzezzhkst
removeRoad;oifinnybjrkspupolm;tdlnytdleb
extendRoute;50;rbmrhkifwsnkxsse
removeRoad;xvekfjqttaewpxkkiocdhhqw;uzdsgvzxsdgfrpmzezzhkst
getRouteDescription;710
addRoad;uzdsgvzxsdgfrpmzezzhkst;jatndbmbfzi;11;7
repairRoad;jatndbmbfzi;rbmrhkifwsnkxsse;-9
getRouteDescription;500
extendRoute;867;qmtshjwkvxrwhhnivgqfpxbj
removeRoad;loimvtfqvodpmfveb;ltokorrvki
addRoad;qmtshjwkvxrwhhnivgqfpxbj;oifinnybjrkspupolm;16;3
getRouteDescription;867
getRouteDescription;867
extendRoute;867;hnsxaeswfw
addRoad;hnsxaeswfw;loimvtfqvodpmfveb;2;1
extendRoute;520;uzdsgvzxsdgfrpmzezzhkst
extendRoute;867;oifinnybjrkspupolm
extendRoute;520;tdlnytdleb
removeRoad;loimvtfqvodpmfveb;ltokorrvki
getRouteDescription;361
getRouteDescription;710
repairRoad;hnsxaeswfw;tdlnytdleb;-12
repairRoad;tdlnytdleb;xvekfjqttaewpxkkiocdhhqw;-8
extendRoute;500;qmtshjwkvxrwhhnivgqfpxbj
repairRoad;oifinnybjrkspupolm;tdlnytdleb;-13
newRoute;506;qmtshjwkvxrwhhnivgqfpxbj;tdlnytdleb
removeRoad;jatndbmbfzi;uzdsgvzxsdgfrpmzezzhkst
getRouteDescription;710
newRoute;61;qmtshjwkvxrwhhnivgqfpxbj;oifinnybjrkspupolm
newRoute;696;jatndbmbfzi;ltokorrvki
repairRoad;rbmrhkifwsnkxsse;oifinnybjrkspupolm;-15
newRoute;326;uzdsgvzxsdgfrpmzezzhkst;hnsxaeswfw
newRoute;954;hnsxaeswfw;hnsxaeswfw
newRoute;812;hnsxaeswfw;xvekfjqttaewpxkkiocdhhqw
getRouteDescription;326
