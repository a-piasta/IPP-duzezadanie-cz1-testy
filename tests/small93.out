1: NIE
2: NIE
3: TAK
4: TAK
5: TAK
6: 
7: NIE
8: NIE
9: NIE
10: TAK
11: NIE
12: TAK
13: NIE
14: NIE
15: TAK
16: NIE
17: NIE
18: TAK
19: TAK
20: NIE
21: 
22: NIE
23: NIE
24: TAK
25: NIE
26: NIE
27: 
28: TAK
29: NIE
30: 
31: NIE
32: TAK
33: TAK
34: 867;ltokorrvki;15;9;qmtshjwkvxrwhhnivgqfpxbj
35: 867;ltokorrvki;15;9;qmtshjwkvxrwhhnivgqfpxbj
36: NIE
37: TAK
38: NIE
39: TAK
40: NIE
41: NIE
42: 
43: 
44: TAK
45: NIE
46: NIE
47: NIE
48: NIE
49: TAK
50: 
51: TAK
52: NIE
53: NIE
54: NIE
55: NIE
56: NIE
57: 
